app/routes/__public/oauth/$id.tsx

```diff
@@ -175,20 +177,73 @@ const loadData = async ({
           response.body.suggestExtension,
         postInstallResult,
       },
-      accessToken: response.body.accessToken,
     };
   }
   return response;
 };
 
+const getAnonymousAccessToken = async ({
+  requestId,
+  params,
+  searchParams,
+}: {
+  requestId: string;
+  params: Record<string, string | undefined>;
+  searchParams: Record<string, string>;
+}) => {
+  const { id = "" } = params;
+  const { code, ...customParams } = searchParams;
+  const cxn = await getMysql(requestId);
+
+  const accessTokenByCode = await cxn
+    .select({ accessToken: accessTokens.value })
+    .from(accessTokens)
+    .where(eq(accessTokens.code, searchParams.code))
+    .execute();
+
+  if (accessTokenByCode.length)
+    return { accessToken: accessTokenByCode[0].accessToken };
```

Comment:
no way to sort? (id/date)
So in rare case of duplicate code and if we don't have a cleaning function, could continually error?

Reply:
let's just stick with [0] for now

Reply:
@dvargas92495 added